src/stcal/resample/utils.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+from stdatamodels.dqflags import interpret_bit_flags
```

Comment:
```suggestion
from stdatamodels.dqflags import interpret_bit_flags
```
Is there a non-stdatamodels way to handle this. We can't make stdatamodels a dependency of this code.

Reply:
yes, there is. I'll fix it (I forgot to switch to `astropy` although `stdatamodels.dqflags.interpret_bit_flags()` for unknown to me reasons does things slightly different. I think _that_ function should be removed altogether from `stdatamodels.dqflags`.

Reply:
oh, I did it in `resample.py` but not in utils.